src/x-platform/index.js

```diff
@@ -1,26 +1,107 @@
+import 'html5-desktop-notifications'
+import {notificationsConfig} from './constants'
+import noop from 'lodash/utility/noop'
+
+
+// Electron brings own require to window
+let electron
+let remote
+if (window.require) {
```

Comment:
instead of this, you can just add external modules to the build and then check if the import is not null.